seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json

```diff
@@ -572,21 +572,26 @@
       "type": "bonfire",
       "title": "Problem 15: Lattice paths",
       "tests": [
-        "assert.strictEqual(euler15(), 137846528820, 'message: <code>euler15()</code> should return 137846528820.');"
+        "assert.strictEqual(latticePaths(4), 70, 'message: <code>latticePaths(4)</code> should return 70.');",
+        "assert.strictEqual(latticePaths(9), 48620, 'message: <code>latticePaths(9)</code> should return 48620.');",
+        "assert.strictEqual(latticePaths(20), 137846528820, 'message: <code>latticePaths(20)</code> should return 137846528820.');"
       ],
-      "solutions": [],
+      "solutions": ["function latticePaths(gridSize) {\n  let paths = 1;\n\n  for (let i = 0; i < gridSize; i++) {\n    paths *= (2 * gridSize) - i;\n    paths /= i + 1;\n  }\n  return paths;\n}"],
       "translations": {},
       "challengeSeed": [
-        "function euler15() {",
+        "function latticePaths(gridSize) {",
         "  // Good luck!",
         "  return true;",
         "}",
         "",
-        "euler15();"
+        "latticePaths(4);",
+        "latticePaths(9);",
```

Comment:
@scissorsneedfoodtoo ideally, we only want to call the challenge function in the seed a maximum of one time

Reply:
@Bouncey, I'm sorry about that. I can go back and change that in this and my other PRs.